README.rst

```diff
@@ -9,3 +9,18 @@ We currently ship:
 
 * An extension for building docs like Read the Docs
 * ``template-meta`` - Allows users to specify template overrides in per-page context.
+
+
+Releasing
+---------
+
+#. Increment the version in ``setup.py``
+#. Tag the release in git: ``git tag $NEW_VERSION``.
+#. Push the tag to GitHub: ``git push --tags origin``
+#. Upload the package to PyPI:
+
+    .. code:: bash
+
+        $ rm -rf dist/
+        $ python setup.py sdist bdist_wheel
+        $ twine upload --sign --identity security@readthedocs.org dist/*
```

Comment:
I wonder if we should include the RTD `common` repo and use the release & changelog scripts from there.